### DISCUSSION TITLE
Errors after installing AzuraCast

### DISCUSSION THREAD

**Discussion by gwbs** [AzuraCast/AzuraCast#6298] [Q&A]
I installed AzuraCast through VirtualBox, but I have several problems: 

1. The IP address that was assigned to me cannot connect. The connection timed out.
2. Bind address already in use
3. Driver failed programming external connectivity on endpoint azuracast
4. Error starting userland proxy

I am not sure what the problem is. I am new to AzuraCast. How can those problems be solved?

**Comment by scysys** [comment]
Maybe your error is your setup related. Check your VM in general.

- Can you connect over SSH to your VM.
- Can you Ping other Services directly from your VM.
- Maybe also Firewall.
